bedrock/firefox/templates/firefox/includes/download-unsupported.html

```diff
@@ -30,7 +30,7 @@
     </p>
 
     <small class="fx-unsupported-message-all-link">
-      <a href="{{ url('firefox.all') }}#product-desktop-esr">
+      <a href="{{ firefox_url('desktop', 'all', 'esr') }}>
```

Comment:
ah found it:

```suggestion
      <a href="{{ firefox_url('desktop', 'all', 'esr') }}">
```
<details><summary>sshot ↓</summary>

![Screen Shot 2024-07-18 at 15 39 15](https://github.com/user-attachments/assets/4c541945-b35c-42f0-8ccf-8ca7eaf3c5b0)

</details>